src/controllers/controller.cpp

```diff
@@ -274,6 +306,10 @@ namespace inria_wbc {
                 error += " (t=" + std::to_string(t_) + ")";
                 throw IWBC_EXCEPTION(error);
             }
+            if (check_model_collisions_)
+                is_model_colliding_ = collision_check_.is_colliding(robot_->model(), tsid_->data());
+            if (is_model_colliding_ && !q_.isZero(1e-3))
+                send_cmd_ = false;
```

Comment:
In verbose mode we could add a print that there is a collision?

Reply:
There  is already this print inside ```CollisionCheck::is_colliding```